iOS/UI/Swipeable Input/SwipeableInputAccessoryView.swift

```diff
@@ -292,117 +292,115 @@ class SwipeableInputAccessoryView: View, AttachmentViewControllerDelegate, UIGes
 
     func attachmentView(_ controller: AttachmentViewController, didSelect attachment: Attachment) {}
 
+    // MARK: - Pan Gesture Handling
```

Comment:
This is the most significant change. You can now drag the preview view around vertically and horizontally.